### DISCUSSION TITLE
[BUG] [SP-API] [Issue]Send to Amazon" prompts failure to update the shipment

### DISCUSSION THREAD

**Discussion by xingxi521** [amzn/selling-partner-api-models#3088] [General]
I used sp-api PutTransportContent to update tracking to shipment,but return error said :
This shipment was created using Send to Amazon and no updates can be made to this shipment.
How should I update tracking to shipment?

**Comment by sungolivia** [comment]
can you change the title to be more descriptive for others to come in and take a look?

**Reply by xingxi521** [reply]
Ok, the title has been changed

**Comment by ouzhiman** [comment]
This shipment was created using Send to Amazon and no updates can be made to this shipment
I also have the same problem, how to solve it?

**Reply by xingxi521** [reply]
I'm still looking for a solution, too

**Reply by ouzhiman** [reply]
hi, did you find a solution?

**Reply by xingxi521** [reply]
Amazon seems to have fixed the problem

**Reply by ouzhiman** [reply]
I still can't submit normally, which API to submit?
"This shipment was created using Send to Amazon and no updates can be made to this shipment"

**Reply by ouzhiman** [reply]
PutTransportContent this?

**Comment by xingxi521** [comment]
yes




星系
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "amzn/selling-partner-api-docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月12日(星期一) 下午2:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [amzn/selling-partner-api-docs] [BUG] [SP-API] [Issue]Send to Amazon" prompts failure to update the shipment (Discussion amzn/selling-partner-api-docs#2731)





 
PutTransportContent this?
